### PR TITLE
Docs: Update column virtualization docs

### PR DESCRIPTION
**Pull request by kirszenbaum**
This PR uses the input from [1457](https://github.com/handsontable/dev-handsontable/issues/1457) to:
- Update the **Column virtualization** guide to reflect the newly-added `renderAllColumns` feature
- Update the **Accessibility** guide with a new section on cells virtualization
- Improve the API ref of `renderAllRows` and `renderAllColumns`

Additionally, this PR:
- Fixes the wrong link text in the `renderAllColumns` API ref ("row" instead of "column")
- Adds a few links to the `renderAllColumns` API ref
- In the API ref, adds links between `renderAllRows` and `viewportRowRenderingOffset`, and between `renderAllColumns` and `viewportColumnRenderingOffset`
- In [Accessibility > Conformance with standards](https://handsontable.com/docs/javascript-data-grid/accessibility/#conformance-with-standards), replaces the awkward double-bullet list with a table

### Updated sections
- http://localhost:8080/docs/javascript-data-grid/column-virtualization/#overview
- http://localhost:8080/docs/javascript-data-grid/api/options/#renderallcolumns
- http://localhost:8080/docs/javascript-data-grid/accessibility/#conformance-with-standards
- http://localhost:8080/docs/javascript-data-grid/api/options/#viewportcolumnrenderingoffset
- http://localhost:8080/docs/javascript-data-grid/api/options/#viewportrowrenderingoffset
- http://localhost:8080/docs/javascript-data-grid/api/options/#renderallrows
- http://localhost:8080/docs/javascript-data-grid/api/options/#renderallcolumns

[skip changelog]

### CODE REVIEW

**Comment by kirszenbaum** [comment]
@evanSe Ready for review, with a few caveats:

- I made an assumption that `renderAllRows` and `renderAllColumns` ovewrite `viewportRowRenderingOffset` and `viewportColumnRenderingOffset`, respectively. If it's not true, let me know.
- Please double check the changes to the demo, especially the React one.
- Shouldn’t DOM virtualization be listed in the Known limitations section of the a11y guide? I didn't add it for now.

File: docs/content/guides/columns/column-virtualization.md [hunk 1]
```diff
@@ -24,7 +24,10 @@ Render hundreds of columns without freezing the browser, using column virtualiza
 ## Overview
 
 To process a large number of columns in a browser Handsontable utilizes the virtualization process to display only the visible part of the grid with a small
-offset for a better scrolling experience. This feature is turned on by default and can be turned off only for rows, not columns.
+offset for a better scrolling experience.
+
+This feature is enabled by default and can be turned off by setting the [`renderAllColumns`](@/api/options.md#renderallcolumns) option to `true`.
+You might want to consider turning it off for [accessibility](@/guides/accessibility/accessibility.md#disabling-dom-virtualization-for-improved-accessibility) reasons.
```
Comment (evanSe) [inline docs/content/guides/columns/column-virtualization.md:30]: This is not needed
```suggestion
```

File: docs/content/guides/rows/row-virtualization.md [hunk 2]
```diff
@@ -26,6 +26,7 @@ Render thousands of rows without freezing the browser, using row virtualization.
 Virtualization allows Handsontable to process hundreds of thousands of records without causing the browser to hang. This feature draws only the visible part of the grid, displaying the minimum items physically rendered in the DOM. The elements outside the viewport are rendered when you scroll across the grid. Depending on your configuration, there might be a small offset of columns or rows rendered outside the viewport to make the scrolling performance smoother.
 
 This feature is enabled by default and can be turned off by setting the [`renderAllRows`](@/api/options.md#renderallrows) option to `true`.
+You might want to consider turning it off for [accessibility](@/guides/accessibility/accessibility.md#disabling-dom-virtualization-for-improved-accessibility) reasons.
```
Comment (evanSe) [inline docs/content/guides/rows/row-virtualization.md:29]: same for this, developer can determine based on requirements.
```suggestion
```

File: docs/content/guides/accessibility/accessibility.md [hunk 3]
```diff
@@ -39,18 +39,15 @@ Accessibility features of Handsontable include:
...
-  - [Section 508 of the US Rehabilitation Act](https://www.section508.gov/)
-  - [Americans with Disabilities Act (ADA)](https://www.ada.gov/resources/web-guidance/)
-
-- **Europe / European Union:**
-  - [European Accessibility Act (EAA)](https://ec.europa.eu/social/main.jsp?catId=1202)
-  - [Web Accessibility Directive (WAD)](https://eur-lex.europa.eu/legal-content/EN/LSU/?uri=CELEX:32016L2102)
-
-- **Canada:**
-  - [Standard on Web Accessibility](https://www.tbs-sct.canada.ca/pol/doc-eng.aspx?id=23601)
+| Region | Standards                                                                                                                                                                                        |
```
Comment (evanSe) [inline docs/content/guides/accessibility/accessibility.md:46]: Is it possible to make the table fullscreen?

**Review by evanSe — APPROVED** [review]

File: docs/content/guides/accessibility/accessibility.md (hunk 3 above)
Comment (kirszenbaum) [reply docs/content/guides/accessibility/accessibility.md:46]: I guess we'd need to write the whole table in HTML, not sure if we want that?

Comment (kirszenbaum) [reply docs/content/guides/accessibility/accessibility.md:46]: Inline HTML works, so we can do this:

```
| <div style="width:365px">Region</div> | Standards |
| --- | --- |                                                                                                                             
```

But this doesn't work, not sure why:

```
| <div style="width:50%">Region</div> | Standards |
| --- | --- |                                                                                                                             
```

Comment (evanSe) [reply docs/content/guides/accessibility/accessibility.md:46]: Lets do that then

**Review by evanSe — DISMISSED** [review]

**Review by evanSe — APPROVED** [review]

**Pull request merged**